packages/socket-graphiql/package.json

```diff
@@ -34,11 +34,12 @@
   ],
   "dependencies": {
     "@absinthe/socket": "0.1.10",
+    "@babel/runtime": "7.2.0",
     "@jumpn/utils-graphql": "0.5.3",
```

Comment:
Could you create a new release of "@jumpn/utils-graphql" with updated dependencies as well? At the moment we would pull in two different versions of graphql (0.11.7 vs. 14.0.2).

Reply:
Sure I will release a new version of **@jumpn/utils-graphql** with the dependencies updated.